modules/pixi/styles.js

```diff
@@ -30,568 +31,529 @@ import { osmPavedTags } from '../osm/tags';
-  },
-  footway: {
-    casing: { width: 5, color: 0xffffff },
-    stroke: { width: 3, color: 0x998888, dash: [6, 6], cap: PIXI.LINE_CAP.BUTT }
-  },
-  crossing_marked: {
-    casing: { width: 5, color: 0xddccaa },
-    stroke: { width: 3, color: 0x4c4444, dash: [6, 3], cap: PIXI.LINE_CAP.BUTT }
-  },
-  crossing_unmarked: {
-    casing: { width: 5, color: 0xddccaa },
-    stroke: { width: 3, color: 0x776a6a, dash: [6, 4], cap: PIXI.LINE_CAP.BUTT }
-  },
-  cycleway: {
-    casing: { width: 5, color: 0xffffff },
-    stroke: { width: 3, color: 0x58a9ed, dash: [6, 6], cap: PIXI.LINE_CAP.BUTT }
-  },
-  bridleway: {
-    casing: { width: 5, color: 0xffffff },
-    stroke: { width: 3, color: 0xe06d5f, dash: [6, 6], cap: PIXI.LINE_CAP.BUTT }
-  },
-  corridor: {
-    casing: { width: 5, color: 0xffffff },
-    stroke: { width: 3, color: 0x8cd05f, dash: [2, 8], cap: PIXI.LINE_CAP.ROUND }
-  },
-  steps: {
-    casing: { width: 5, color: 0xffffff },
-    stroke: { width: 3, color: 0x81d25c, dash: [3, 3], cap: PIXI.LINE_CAP.BUTT }
-  },
-
-  river: {
-    casing: { width: 10, color: 0x444444 },
-    stroke: { width: 8, color: 0x77dddd }
-  },
-  stream: {
-    casing: { width: 7, color: 0x444444 },
-    stroke: { width: 5, color: 0x77dddd }
-  },
-  ridge: {
-    stroke: { width: 2, color: 0x8cd05f}  // rgb(140, 208, 95)
-  },
-
-  runway: {
-    casing: { width: 10, color: 0x000000, cap: PIXI.LINE_CAP.BUTT },
-    stroke: { width: 8, color: 0xffffff, dash: [24, 48], cap: PIXI.LINE_CAP.BUTT }
-  },
-  taxiway: {
-    casing: { width: 7, color: 0x444444 },
-    stroke: { width: 5, color: 0xffff00 }
-  },
-
-  railway: {
-    casing: { width: 7, color: 0x555555, cap: PIXI.LINE_CAP.BUTT },
-    stroke: { width: 2, color: 0xeeeeee, dash: [12, 12], cap: PIXI.LINE_CAP.BUTT,  }
-  },
-
-  ferry: {
-    casing: { alpha: 0 },  // disable
-    stroke: { width: 3, color: 0x58a9ed, dash: [12, 8], cap: PIXI.LINE_CAP.BUTT }
-  },
-
-  boundary: {
-    casing: { width: 6, color: 0x82b5fe, cap: PIXI.LINE_CAP.BUTT },
-    stroke: { width: 2, color: 0xffffff, dash: [20, 5, 5, 5], cap: PIXI.LINE_CAP.BUTT }
-  },
-  boundary_park: {
-    casing: { width: 6, color: 0x82b5fe, cap: PIXI.LINE_CAP.BUTT },
-    stroke: { width: 2, color: 0xb0e298, dash: [20, 5, 5, 5], cap: PIXI.LINE_CAP.BUTT }
-  },
-
-  barrier: {
-    casing: { alpha: 0 },  // disable
-    stroke: { width: 3, color: 0xdddddd, dash: [10, 5, 1, 5], cap: PIXI.LINE_CAP.ROUND }
-  },
-  barrier_wall: {
-    casing: { alpha: 0 },  // disable
-    stroke: { width: 3, color: 0xdddddd, dash: [10, 5, 1, 5], cap: PIXI.LINE_CAP.ROUND }
-  },
-  barrier_hedge: {
-    fill:   { color: 0x8cd05f, alpha: 0.3 },   // rgb(140, 208, 95)
-    casing: { alpha: 0 },  // disable
-    stroke: { width: 3, color: 0x8cd05f, dash: [10, 5, 1, 5], cap: PIXI.LINE_CAP.ROUND }
-  },
-
-  tree_row: {
-    casing: { width: 7, color: 0x444444 },
-    stroke: { width: 5, color: 0x8cd05f }
+export class STYLES {
```

Comment:
Also, class names should be CamelCase- so let's rename this to `Styles`, which should further reinforce that we're no longer dealing with a blob of statically defined stuff.

Reply:
Done!